### ISSUE TITLE
[SF][0.23.2][MessageBox]MessageBox header icon should have role="presentation" and aria-hidden="true"

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents-react#3014]
**Describe the bug**
Hi Team,
When using MessageBox set type Success, the header icon will render svg with role="img" and aria-label, witch will be announced  using screen reader.

**To Reproduce**
1. Go to this [link](https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--default-story)
2. set type Success, and Click on "Open Messagebox" button.
3. check the header icon dom element using chrome dev tool.

**Expected behavior**
Header icon should have role="presentation" and aria-hidden="true"

**Screenshots**
<img width="1463" alt="Screen Shot 2022-06-01 at 17 11 56" src="https://user-images.githubusercontent.com/78520316/171370174-da56074b-75d8-40b7-820a-92ea1d398239.png">
**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.3.1
`@ui5/webcomponents-react` version: 0.23.2
Operating System: All
Browser: All
